perftests/standalone-enc/fd_drain.c

```diff
@@ -0,0 +1,63 @@
+#include <stdint.h>
+#include <unistd.h>
+
+#include "fork_func.h"
+#include "warnp.h"
+
+#include "fd_drain.h"
+
+/**
+ * fd_drain(fd_p):
+ * Drain bytes from the file descriptor ${*fd_p} -- passed as an (int *) -- as
+ * quickly as possible.
+ */
+int
+fd_drain(void * cookie)
+{
+	int fd = *((int *)cookie);
+	uint8_t mybuf[FD_DRAIN_MAX_SIZE];
+	ssize_t readlen;
+
+	/* Loop until we hit EOF. */
+	do {
+		/*
+		 * This will almost always read 1060 bytes (size of an
```

Comment:
This comment seems like it made sense in the original context of where this code was used, but doesn't belong any more once `fd_drain` is in a separate file since it could end up being used by completely different callers.